db/seeds.rb

```diff
@@ -3,9 +3,11 @@
 TEST_PASSWORD = '1234567'
 TEST_EMAIL = 'a@a.a'
 
-## Landing pages
-landing = Landing.find_or_create_by!(slug: 'test-landing', home_sort_order: 0)
-topic = LandingTopic.find_or_create_by!(title: 'Test Topic', landing: landing)
+if Landing.count < 1
```

Comment:
```suggestion
if Landing.none?
```
Ça évite de les compter tous :)

Reply:
je l'ai cherché pourtant cette méthode, je savais qu'elle existait mais impossible de me rappeler du nom 😄